NomaiVR/Input/VRActionInput.cs

```diff
@@ -52,8 +51,7 @@ public string[] GetText()
             {
                 return new string[] { };
             }
-            var holdInputText = GetHoldActionText();
-            return holdInputText.Add(WrapWithBrackets(result)).ToArray();
+            return GetHoldActionText().Concat(new[] { WrapWithBrackets(result) }).ToArray();
```

Comment:
I think you said you would revert these since they're out of scope? But if you're sure the result is the same, I'm fine with it. I'm just always nervous about the input stuff since it has always been super fragile :P